region/maridia/inner-pink.json

```diff
@@ -1317,86 +830,10 @@
                       {"canShineCharge": {
```

Comment:
At 826/827 there's "or" & "and" without options.

Reply:
Yup. Removed a bunch of stuff there and missed the cleanup.